tests/test_cli.py

```diff
@@ -132,26 +128,6 @@ def test_before(capfd):
     with tempfile.TemporaryDirectory() as tmp_dir:
         shutil.copytree(project_dir, tmp_dir, dirs_exist_ok=True)
 
-        captured = _run_fal(
```

Comment:
why are you removing this test? I think the idea is to test before WITH selection

Reply:
Nothing is supposed to be printed in this case actually.

Reply:
Maybe we need to add new scripts to better test these

Reply:
The test below this tests exactly that. This test was unnecessary.

Reply:
Actually sure, we can make sure that nothing gets printed.